.travis.yml

```diff
@@ -1,30 +1,23 @@
 sudo: false
 language: python
-python:
-    - "2.7"
-    - "3.4"
```

Comment:
To be clearer: we can either control the python version at the Travis level, or the tox level, but not both. I'm open to doing either :-)